### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by striezel**
This pull request fixes some typos, no functional changes intended.

### CODE REVIEW

**Comment by Try** [comment]
Hi, @striezel and thanks for PR!

Path, looks good, can merge after resolving conflict in `game/resources.cpp`

File: game/graphics/mesh/animationsolver.cpp [hunk 1]
```diff
@@ -240,7 +240,7 @@ const Animation::Sequence* AnimationSolver::implSolveAnim(AnimationSolver::Anim
       s = solveFrm("T_%sPARADEJUMPB",st);
     if(s!=nullptr)
       return s;
-    // This is bases on original game: if no move-back animation, even in water, game defaults to standart walk-bak
+    // This is bases on original game: if no move-back animation, even in water, game defaults to standard walk-bak
```
[the remark this answers is not in the document]
Comment (striezel) [reply game/graphics/mesh/animationsolver.cpp:243]: Good catch! :+1:

**Comment by striezel** [comment]
@Try
> Path, looks good, can merge after resolving conflict in `game/resources.cpp`

The force push to the `master` branch caused the conflict. You should not really force push to the most important branch of a repo, especially when other people have already based their work on it.

However, I've resolved the conflict, so this can now be merged cleanly.

**Comment by Try** [comment]
> However, I've resolved the conflict, so this can now be merged cleanly.

Yeah, sorry for that. Merged now!

**Pull request merged**